test/acceptance/features/steps/util.py

```diff
@@ -1,5 +1,10 @@
 import os
-
+from string import Template
 
 def scenario_id(context):
     return f"{os.path.basename(os.path.splitext(context.scenario.filename)[0]).lower()}-{context.scenario.line}"
+
+def get_scenario_id(context, text=None):
+    if text is None:
```

Comment:
instead of checking if text is none, you can set the default value to "$scenario_id" in the signature already.

Reply:
Fixed.